javascript/javascript2/week3/warmup.js

```diff
@@ -0,0 +1,93 @@
+
+window.addEventListener("load", () => {
+  setTimeout(() => {
+    console.log("load efter 2,5 seconds");
+  }, 2500);
+});
+
+// function with delay parameter
+let delayfunction = (stringToLog,delay) =>{
+setTimeout(() => {
+    console.log(stringToLog)
+}, delay);
+}
+
+const btn1 = document.getElementById('btn1');
+btn1.onclick = () => {
+delayfunction("this string logget after 2 seconds of btn clicked   ", 2000)};
```

Comment:
Did you test this to make sure it works? The method you're looking for to listen for a click is `addEventListener`

Reply:
Hi Braedon . i have tested it with onclick. and with addEventListener. both events make same result